packages/deploy-tooling/src/base/archive.ts

```diff
@@ -0,0 +1,22 @@
+import type { ZipFile } from 'yazl';
```

Comment:
The name archive doesn't reflect what createBuffer does.

Reply:
All files are structured in three main modules `cli`, `ui5` and `base` (there is also `types` but that is standard independent of the structure here). The base module then offers the three sub-modules `archive`, `config`, `deploy` and `prompt`. The other two optionally extend each of the sub-modules.
I think this is pretty well structure and is supposed to make it easy for a ready to find code.
Is the term `archive` problematic for you? If yes, what would be the alternative for the submodule?

Reply:
I don't have problem with the code structure. Only the file name archive. It is not a big issue. Does it make sense to move `createArchiveFromFolder` and `getFileNames` from cli/archive.ts to base/archive.ts. They are quite fit for 'base' blocks. Then I am convinced with 3 archive.ts names:

- base/archive.ts has base implementation to support zip file
- ui5/archive.ts provides specific archive for ui5 core task
- cli/archive.ts utilizes base/archive.ts to handle deploy of archive from multiple sources.

Reply:
I have moved `getFileNames` as suggested to `base` however, I kept `createArchiveFromFolder` where it is because now, `cli/archive` has a very consistent abstraction level of all functions.
https://github.com/SAP/open-ux-tools/pull/690/commits/a8fbc77f2658df658c967587fd40eab88974fe4c